src/libraries/System.Private.Runtime.InteropServices.JavaScript/tests/System.Private.Runtime.InteropServices.JavaScript.Tests.csproj

```
@@ -18,13 +26,28 @@
     <Compile Include="System\Runtime\InteropServices\JavaScript\Http\HttpRequestMessageTest.cs" />
     <Compile Include="System\Runtime\InteropServices\JavaScript\ParallelTests.cs" />
     <Compile Include="System\Runtime\InteropServices\JavaScript\TimerTests.cs" />
+    <Compile Include="System\Runtime\InteropServices\JavaScript\UriMarshaller.cs" />
+    <EmbeddedResource Include="System\Runtime\InteropServices\JavaScript\TestVector3Marshaller.js">
+      <LogicalName>TestVector3Marshaller.js</LogicalName>
+    </EmbeddedResource>
+    <EmbeddedResource Include="System\Runtime\InteropServices\JavaScript\UriMarshaller.js">
+      <LogicalName>UriMarshaller.js</LogicalName>
+    </EmbeddedResource>
+    <None Include="$(IntermediateOutputPath)\generated\JavaScript.MarshalerGenerator\System.Runtime.InteropServices.JavaScript.JsImportGenerator\JsImport.g.cs" />
```

Last comment from me :)

```suggestion
    <None Include="$(CompilerGeneratedFilesOutputPath)\JavaScript.MarshalerGenerator\System.Runtime.InteropServices.JavaScript.JsImportGenerator\JsImport.g.cs" />
```